packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/index.js

```diff
@@ -142,6 +64,11 @@ export default class DimensionSwitcher extends PureComponent {
         loadingPresets: {}
     };
 
+    getDimensionIcon = (dimensionName, contentDimensionsObject) => {
+        const dimensionConfiguration = contentDimensionsObject[dimensionName];
+        return $get('icon', dimensionConfiguration) && $get('icon', dimensionConfiguration);
```

Comment:
I don't "$get" this line?

Reply:
It is there since 7 years... https://github.com/neos/neos-ui/blob/e4080d6bb1a14a2c5eb843109ee82af38b84c8cf/packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/index.js#L250

Just moved it out and did not think about it that much. But will check that and adapt it.

Reply:
Yeah didn't check whether each part was new. But if we anyway touch it we can improve the internals a bit.

Reply:
This is gone now ;)